internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/DaVinciClientTest.java

```diff
@@ -1004,71 +1001,58 @@ public void testReadCompressedData(CompressionStrategy compressionStrategy) thro
    * For the local P2P testing, need to setup two different directories and ports for the two Da Vinci clients in order
    * to avoid conflicts.
    */
-  @Test(timeOut = 2 * TEST_TIMEOUT, enabled = false)
+  @Test(timeOut = 2 * TEST_TIMEOUT)
   public void testBlobP2PTransferAmongDVC() throws Exception {
+    String baseDataPath = Utils.getTempDataDirectory().getAbsolutePath();
+    String zkHosts = cluster.getZk().getAddress();
+    int port1 = TestUtils.getFreePort();
+    int port2 = TestUtils.getFreePort();
     Consumer<UpdateStoreQueryParams> paramsConsumer = params -> params.setBlobTransferEnabled(true);
     String storeName = Utils.getUniqueString("test-store");
     setUpStore(storeName, paramsConsumer, properties -> {}, true);
-    String dvcPath1 = Utils.getTempDataDirectory().getAbsolutePath();
-    // backend config for first DVC client
-    int port1 = TestUtils.getFreePort();
+    LOGGER.info("Port1 is {}, Port2 is {}", port1, port2);
+    LOGGER.info("zkHosts is {}", zkHosts);
+
+    // Start the first DaVinci Client using DaVinciUserApp for regular ingestion
+    ForkedJavaProcess forkedDaVinciUserApp1 = ForkedJavaProcess.exec(
+        DaVinciUserApp.class,
+        zkHosts,
+        baseDataPath,
+        storeName,
+        "100",
+        "10",
+        "false",
+        Integer.toString(port1),
+        Integer.toString(port2));
+
+    // Wait for the first DaVinci Client to complete ingestion
+    Thread.sleep(60000);
```

Comment:
IMO, we don't need this. We can check whether the following statements are met or not, with a timeout
```
    for (int i = 0; i < 3; i++) {
      String snapshotPath = RocksDBUtils.composeSnapshotDir(dvcPath1 + "/rocksdb", storeName + "_v1", i);
      Assert.assertTrue(Files.exists(Paths.get(snapshotPath)));
    }
```

If yes, it mean the batch ingestion has done and snapshots are created already

Reply:
implemented in latest commit